strategy/live_trader.py

```
@@ -1041,6 +1101,45 @@ def update_config(self, offset: Optional[float] = None,
+                            f"(status={m.status}, filled_up={m.filled_up}, filled_down={m.filled_down}). "
+                            f"Wait for window settlement or stop exit before deselecting."
+                        )
+
+                for s in to_remove:
+                    m = self.markets[s]
+                    self.historical_realized_pnl += m.realized_pnl_usd
+                    for oid in (m.order_id_up, m.order_id_down, m.next_order_id_up, m.next_order_id_down, m.order_id_exit_up, m.order_id_exit_down):
+                        if oid:
+                            try:
+                                self.cancel_live_order(oid)
+                            except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)